types/react-virtualized-select/index.d.ts

```diff
@@ -22,9 +23,13 @@ export interface VirtualizedOptionRenderOptions<T> {
     valueArray: T[];
 }
 
+interface OptionHeightFunc {
+    (props: any): number;
+  }
+
 export interface AdditionalVirtualizedSelectProps<TValue> {
     maxHeight?: number;
-    optionHeight?: number;
+    optionHeight?: number | OptionHeightFunc;
```

Comment:
Per [the docs](https://github.com/bvaughn/react-virtualized-select#react-virtualized-select-props), it looks like this signature can be a bit more precise. I think it can be typed `({ option: TValue }) => number`, or at least `({ option: any }) => number`.

Also, could you just inline the definition here? The interface adds a lot of indirection.

Reply:
That does look like the signature of the interface specified here. The body of OptionHeightFunc is:

        (props: any): number;

Reply:
I think I purposefully chose not to do `({ option: TValue }) => number` as it limits the function to having to take an object of that shape as args. In our case, we were wanting to pass in other arguments which I think makes sense as it's our callback function we should control what we do with it as long as it returns a number. I won't be able to check my code to confirm until Monday now though as it's on my work machine.

Reply:
> Are you suggesting props is tightened up with { option: any} in that interface?

Yes, and also that the interface be removed because being a simple non-overloaded one-argument non-generic function, putting it in an interface seems like extra indirection for no value.

> it limits the function to having to take an object of that shape as args.

Your function can choose not to take any arguments, but if it takes one, it has to be `{ option: TValue }` as far as I can tell from the docs. How would you pass in different arguments if react-virtualized-select is the one calling your function?